bin/checkver.ps1

```diff
@@ -1,18 +1,38 @@
-# checks websites for newer versions using an (optional) regular expression defined in the manifest
-# use $dir to specify a manifest directory to check from, otherwise ./bucket is used
+<#
+.SYNOPSIS
+    Check manifest for a newer version.
+.DESCRIPTION
+    Checks websites for newer versions using an (optional) regular expression defined in the manifest.
+.PARAMETER App
+    Manifest name to search.
+    Placeholders are supported.
+.PARAMETER Dir
+    Where to search for manifest(s).
+.PARAMETER Update
+    Update given manifest
+.PARAMETER ForceUpdate
+    Update given manifest(s) even when there is no new version.
+    Useful for hash updates.
+.PARAMETER SkipSupported
+    Updated manifests will not be shown.
+.EXAMPLE
+    PS $BUCKETDIR $ .\bin\checkver.ps1
+    Check all manifest inside default directory.
+#>
 param(
-    [String]$app,
-    [String]$dir,
-    [Switch]$update = $false,
-    [Switch]$forceUpdate = $false
+    [String] $App = '*',
+    [ValidateScript( {
+        if (!(Test-Path $_ -Type Container)) {
+            throw "$_ is not a directory!"
+        }
+        $true
+    })]
+    [String] $Dir = "$psscriptroot\..\bucket",
+    [Switch] $Update,
+    [Switch] $ForceUpdate,
+    [Switch] $SkipSupported
```

Comment:
`SkipUpdated` or `SkipUpdateToDate` would be more descriptive?

Reply:
I used `SkipSuported` as in https://github.com/lukesampson/scoop/blob/master/bin/missing-checkver.ps1#L4

Reply:
But it means something different in this case. All these apps support the autoupdate feature.

Reply:
There could be set Alias as:

```ps
[Alias("SkipUpdated", ...)]
$SkipSupported
```